src/plugins/kibana/public/management/styles/main.less

```diff
@@ -55,16 +79,6 @@ kbn-management-landing {
     position: absolute;
   }
 
-  .panel-heading i {
```

Comment:
I created the `panel-heading-icon` class to avoid having this high-specificity child selector. Now these styles become more readable because the selector has a meaningful name, and also more flexible because we can apply it to a `div` or `span` should we decide to change up our icon system.